ui/src/app/store/face-recognition/effects.ts

```diff
@@ -65,6 +65,22 @@ export class FaceRecognitionEffects {
     })
   );
 
+  @Effect({ dispatch: false })
+  SignUpSuccess: Observable<any> = this.actions.pipe(
```

Comment:
please follow the effects naming convention. they should start from the lower case and have $ sign at the end
signUpSuccess$